mrt/static/js/printouts.js

```diff
@@ -21,7 +21,13 @@ $(function () {
         var job_row = $('#' + this.id);
         job_row.addClass('success');
         job_row.find('.status').text(STATUS.FINISHED);
-        var html = $('<b>').text('Success - please refresh');
+        
+        var filePath = "[?:[a-zA-Z0-9-_\.]+(?:.xls|.xlsx)";
+        
+        var html = 'Success';
```

Comment:
You should be able to put the custom message returned by the function here instead. (i.e. the one from `result`)